smqtk_iqr/web/search_app/modules/iqr/iqr_search.py

```diff
@@ -150,14 +150,15 @@ def __init__(
         self._static_data_dir = osp.join(self.work_dir, 'static')
 
         # Custom static host sub-module
-        self.mod_static_dir = StaticDirectoryHost('%s_static' % self.name,
+        self.mod_static_dir = StaticDirectoryHost('%s_static' % self.name.replace('.', '-'),
```

Comment:
This property is, in this case, the value set to `import_name=__name__` above, so we could instead do the replace there once instead of multiple times here.